### PR TITLE
LPS-198436 | master

### PR DESCRIPTION
**Pull request by davisaints**
**JIRA Ticket: [LPS-198436](https://liferay.atlassian.net/browse/LPS-198436)**

### CODE REVIEW

**Comment by davisaints** [comment]
ci:test:sf

**Comment by davisaints** [comment]
Already tested in the current test cases we have.

File: modules/apps/object/object-test/src/testFunctional/macros/ObjectAdmin.macro [hunk 1]
```diff
@@ -400,6 +400,17 @@ definition {
...
+		JSONObject.addObjectValidation(
+			errorMessage = ${errorMessage},
+			labelName = ${labelName},
+			objectName = ${objectName},
+			script = ${script},
+			userEmailAddress = ${userEmailAddress},
+			userPassword = ${userPassword},
+			validationType = ${validationType});
+	}
+
```
Comment (patriciajeanperez) [inline modules/apps/object/object-test/src/testFunctional/macros/ObjectAdmin.macro:413]: Hey @davisaints! I know we have been following this pattern already, but we would like to move away from creating a macro just to call another macro. Our tests can just directly call `JSONObject.addObjectValidation`. Let me know what you think. Thanks!

**Review by patriciajeanperez — CHANGES_REQUESTED** [review]
Great job writing this JSON macro btw! @davisaints

File: modules/apps/object/object-test/src/testFunctional/macros/ObjectAdmin.macro (hunk 1 above)
Comment (davisaints) [reply modules/apps/object/object-test/src/testFunctional/macros/ObjectAdmin.macro]: Looks good, I didn't know we had stopped following this pattern.

Comment (davisaints) [reply modules/apps/object/object-test/src/testFunctional/macros/ObjectAdmin.macro]: Looks good!

File: modules/apps/object/object-test/src/testFunctional/macros/json/JSONObject.macro [hunk 2]
```diff
@@ -598,6 +598,55 @@ definition {
...
+		if (!(isSet(userPassword))) {
+			var userPassword = "test";
+		}
+
+		if (!(isSet(errorMessage))) {
+			var errorMessage = "Please enter a valid entry.";
+		}
+
+		var single_quote = "'";
+		var scape_quotes = "'\''";
```
Comment (patriciajeanperez) [inline modules/apps/object/object-test/src/testFunctional/macros/json/JSONObject.macro:624]: @davisaints did you mean escape_quotes? sorry I missed this in the first review!

**Comment by davisaints** [comment]
ci:forward

**Comment by davisaints** [comment]
The team has already fixed the macro. I just added a necessary parameter that they added in the test. Than you for the review!

cc/ @patriciajeanperez

**Pull request closed without merging**